src/lib/ProductCard/productCard.module.scss

```diff
@@ -0,0 +1,151 @@
+@import '../variables';
+
+.product-card-container {
+  border: 1px solid #f1f1f2;
+  width: 380px;
+
+  @media screen and (max-width: $tablet) {
+    width: 315px;
+  }
+}
+
+.price-container {
+  padding-top: 11px;
+  padding-left: 10px;
+  display: flex;
+}
+
+.product-image {
+  position: relative;
+
+  img {
+    width: 100%;
+  }
+
+  @media screen and (max-width: $mobile) {
+    img {
+      width: 151px;
+      height: 124px;
+    }
+  }
+}
+
+.match-banner {
+  position: absolute;
+  background-color: $gray-11;
+  color: #f4f4f4;
+  padding: 5px;
+  font-size: 14px;
+  font-weight: bolder;
+  line-height: 21px;
+}
+
+.contentRow {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+
+  @media screen and (min-width: $mobile) {
```

Comment:
`min-width` might not work how you expect it to here since that variable works with max-width. I could be wrong. Or we might need to make 2 variables. For example 768< and above works on desktop. 768 and below will work for mobile/tablet